modules/telemetry/v1/data_block.py

```diff
@@ -401,12 +403,13 @@ def __init__(self, mission_time: int, x_axis: int, y_axis: int, z_axis: int) ->
             x_axis: The velocity about the x axis in degrees per second.
             y_axis: The velocity about the y axis in degrees per second.
             z_axis: The velocity about the z axis in degrees per second.
-
+            magnitude: The magnitude of the angular velocity in degrees per second.
         """
         super().__init__(mission_time)
         self.x_axis: int = x_axis
         self.y_axis: int = y_axis
         self.z_axis: int = z_axis
+        self.magnitude: int = round((abs(x_axis) ** 2 + abs(y_axis) ** 2 + abs(z_axis) ** 2) ** 0.5, 2)
```

Comment:
Don't typedef as `int`, I believe this is a float from your test implementation.